charts/sonatype-nexus/templates/proxy-svc.yaml

```diff
@@ -44,4 +44,7 @@ spec:
   {{- if and (eq .Values.nexus.service.type "ClusterIP") .Values.nexus.service.clusterIP }}
   clusterIP: {{ .Values.nexus.service.clusterIP }}
   {{- end }}
+  {{ if eq .Values.nexus.service.type "LoadBalancer" }}
```

Comment:
Should we also check for the service.loadBalancerIP to have a value?

Reply:
I have added the check and also added the whitespace manipulator to maintain consistency with the Cluster IP setting.